src/mscorlib/model.xml

```diff
@@ -683,6 +683,9 @@
       <Member Name="GetHashCode(T)" />
       <Member MemberType="Property" Name="Default" />
     </Type>
+    <Type Name="System.Collections.Generic.EnumEqualityComparer&lt;T&gt;">
```

Comment:
I see.

> EnumEqualityComparer's GetObjectData uses SerializationInfo.SetType 

It is quirk for desktop cross-version compat. The side-effect of this quirk means that serialization roundtrip will give you less performant comparer. Since the serialization is not cross-version compatible in .NET Core, do we want to remove this quirk?